src/gui/imageImportDialog.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <QDialog>
+#include <QDialogButtonBox>
+#include <QTabWidget>
+
+namespace vtfview
+{
+	class ImageImportDialog : public QDialog {
+		Q_OBJECT;
+
+	public:
+		explicit ImageImportDialog(QWidget* pParent = nullptr);
+
+	private:
+		QDialogButtonBox* buttonBox;
+	};
+} // namespace vtfview
```

Comment:
`imageImportDialog.hpp` and `imageImportDialog.cpp` look like they're unused. Let's just remove them for now.